members/MNL000002.yaml

```diff
@@ -48,9 +48,9 @@ contact_form:
             max_length: 3500
     - click_on:
         - value: Submit
-          selector: button[type='submit']
+          selector: "div.col-md-12 button.btn"
```

Comment:
I don't think it warrants any changes here this time, but I do think when possible we should try to avoid selectors that are tied to a specific layout.  For example, in this case if they decide tomorrow they'd like a slightly smaller column layout like col-md-8 (that's probably bootstrap or a similar css library), we'd have to go through and change every one of these again.  When possible we should probably try to target a specific button ID, but they didn't give it one here.  And otherwise, try to target a specific class applied only to that button.  We go through the same thought process when constructing selectors for Galactus tests, so we don't have to go back and update tests every time someone makes a minor cosmetic change to a feature.  

All that being said, I'm still new to all of this.  Does it happen often that we have to change yamls based on cosmetic changes the targets make to their forms?  Or am I over-estimating the frequency at which these forms get changed?  For any of these PRs, definitely free to push back on any feedback if you have past experience that tells you otherwise.